Dockerfile bug

Error in Dockerfile in Travis-CI and DockerHub when build

```
(...)
The command '/bin/sh -c go install [secure]/brcep' returned a non-zero code: 1
Makefile:24: recipe for target 'build.docker' failed
make: *** [build.docker] Error 1
The command "make build.docker" failed and exited with 2 during .
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,5 @@
 FROM golang:alpine
 ADD . /go/src/leogregianin/brcep
-RUN go install leogregianin/brcep
+RUN cd /go/src/leogregianin/brcep && go install
 CMD ["/go/bin/brcep"]
 EXPOSE 8000

